docs/pages/server-access/guides/openssh.mdx

```diff
@@ -10,7 +10,12 @@ accept SSH certificates dynamically issued by a Teleport CA.
 
 Using Teleport and OpenSSH has the advantage of getting you up
 and running, but in the long run, we would recommend replacing `sshd` with `teleport`.
-We've outlined these reasons in [OpenSSH vs Teleport SSH for Servers?](https://goteleport.com/blog/openssh-vs-teleport/)
+`teleport` SSH servers have support for multiple features that are incompatible with OpenSSH:
+
+- [Dynamically updating node labels from the output of configured commands](../../management/admin/labels.mdx)
```

Comment:
```suggestion
- RBAC and resource filtering based on [dynamically updated labels](../../management/admin/labels.mdx)
```
Suggested so there's a clear benefit to this

Reply:
Thanks, done